src/scss/components/alert/index.scss

```diff
@@ -1,35 +1,35 @@
 hx-alert {
-  background-color: $gray-400;
-  color: $gray-1000;
+  background-color: var(--hxAlert, $gray-400);
+  color: var(--hxColor, $gray-1000);
   display: block;
   font-weight: 300;
   line-height: 1.25rem;
 
   &[type="info"] {
-    background-color: $blue-100;
-    color: $gray-1000;
+    background-color: var(--hxBlue, $blue-100);
+    color: var(--hxColor, $gray-1000);
   }
 
   &[type="error"] {
-    background-color: $red-900;
-    color: $gray-0;
+    background-color: var(--hxRed, $red-900);
```

Comment:
Please see above examples to update changes here and below.

Reply:
Changes have been done,Please review